### ISSUE TITLE
unexpected warning message.

### ISSUE THREAD

**Issue by doutriaux1** [UV-CDAT/vcs#309]
@scottwittenburg since your bg fix we now get this message:

```python
import vcs
bg=False
canvas = vcs.init(geometry={"width":1200, "height":800}, bg=bg)
canvas.plot([1,2,3,4])
canvas.png("warn")
```
returns
```
/Users/doutriaux1/anaconda2/envs/nightly2/lib/python2.7/site-packages/vcs/VTKPlots.py:1274: UserWarning: You are saving to png of size different from the current canvas.
It is recommended to set the windows size before plotting or at init time.
This will lead to faster execution as well.
e.g
x=vcs.init(geometry=(1200,800))
#or
x=vcs.init()
x.geometry(1200,800)
```

with bg=True the message does not show

**Comment by scottwittenburg** [comment]
I cannot reproduce this with the latest master `vcs`, whether I set `bg=True` or `bg=False`.  Maybe I have some other difference in my environment though?  Based on the changes I made, that warning should not be showing up given the snippet you provided, and for me, it does not.

Just for some extra discussion however, this warning was in the code before, and I felt there was a case where it made sense to show it.  This case is if you init with one size, then try to save a png with a different size.  The code will note the current plot size, but resize it for the png save, then resize again back to the original size.  It will also show that warning to let you know about the extra renders needed to change size and change back again.
